tests/director/tests/application/application_templates_api_test.go

```diff
@@ -1697,33 +1963,53 @@ func TestRegisterApplicationFromTemplateWithTemplateID(t *testing.T) {
 	//GIVEN
 	ctx := context.Background()
 	appTemplateName := fixtures.CreateAppTemplateName("template")
+	appTemplateName2 := fixtures.CreateAppTemplateName("template-1")
```

Comment:
Regarding L1939: maybe here oauthGraphQLClient should be used consistent with the other requests for create app template.